force-app/main/default/lwc/paymentModalThirdScreen/paymentModalThirdScreen.css

```diff
@@ -0,0 +1,3 @@
+.message {
+  font-size: 1rem;
```

Comment:
i lean towards using OOTB slds classes when possible, and since this is not the most complex custom styling you could use a standard class instead of this CSS: https://www.lightningdesignsystem.com/utilities/text/